README.md

```diff
@@ -8,12 +8,15 @@ This is a port of the [Typesafe Config](https://github.com/typesafehub/config) l
 The library provides Ruby support for the [HOCON](https://github.com/typesafehub/config/blob/master/HOCON.md) configuration file format.
 
 
-At present, it supports supports parsing and modification of existing HOCON/JSON files via the `ConfigFactory` class and the `ConfigValueFactory` class, and rendering parsed config objects back to a String.
+At present, it supports supports parsing and modification of existing HOCON/JSON files via the `ConfigFactory`
+class and the `ConfigValueFactory` class,and rendering parsed config objects back to a String.
 It also supports the parsing and modification of HOCON/JSON files via `ConfigDocumentFactory`.
```

Comment:
The Basic Usage section below has couple examples including the requires. Is that what you're looking for?